src/meta/src/hummock/manager/mod.rs

```diff
@@ -1396,10 +1588,13 @@ where
                         .unwrap()
                         .1
                         .clone()],
+                    all_table_ids,
```

Comment:
Why need to send table ids periodically to CNs?

Reply:
> Why need to send table ids periodically to CNs?

CN needs to use it to delete inexistent index in `CompactionGroupClient`